src/main/webapp/WEB-INF/tags/instructor/search/searchPageInput.tag

```diff
@@ -32,20 +32,20 @@
                 class="glyphicon glyphicon-info-sign">
             </span>
           </li>
+          <li>
+            <input id="students-check" type="checkbox"
+                   name="<%=Const.ParamsNames.SEARCH_STUDENTS%>" value="true"
+                   <c:if test="${data.searchForStudents || !data.searchCommentForResponses}">checked</c:if>>
+            <label for="students-check">
+              Students
+            </label>
+          </li>
           <li>
             <input id="comments-for-responses-check" type="checkbox"
                 name="<%=Const.ParamsNames.SEARCH_COMMENTS_FOR_RESPONSES%>" value="true"
```

Comment:
Update the data in `SEARCH_COMMENTS_FOR_RESPONSES` from `searchcommentforresponses` to `searchquestionresponsesandcommentsonresponses` ?

Reply:
Likewise, `SEARCH_SESSION_FEEDBACK`?

Reply:
`SEARCH_FEEDBACK_SESSIONS_DATA = "searchfeedbacksessionsdata"`

Reply:
Updated